emails/views.py

```diff
@@ -617,17 +620,46 @@ def _sns_message(message_json):
         wrapped_text = relay_header_text + text_content
         message_body["Text"] = {"Charset": "UTF-8", "Data": wrapped_text}
 
-    to_address = user_profile.user.email
-    formatted_from_address = generate_relay_From(from_address, user_profile)
-    response = ses_relay_email(
-        formatted_from_address,
-        to_address,
-        subject,
-        message_body,
-        attachments,
-        mail,
-        address,
+    resender_headers_flag, _ = Flag.objects.get_or_create(
+        name=RESENDER_HEADERS_FLAG_NAME,
+        defaults={"note": "MPP-2117: Filter-friendly mail headers"},
     )
+    resender_headers = (
+        resender_headers_flag.is_active_for_user(user_profile.user)
+        or resender_headers_flag.everyone
```

Comment:
this still boggles my mind that `is_active_for_user` doesn't return `True` when the flag is set to everyone.